tests/test_noninteractive_mode.py

```diff
@@ -28,6 +26,48 @@ def test_query(self, query_str, file_baseline):
         output_baseline = self.get_file_contents(file_baseline)
         assert output_query == output_baseline
 
+    def test_long_query(self):
+        """ Special test that inputs a SQLCMD function before outputting a large query.
+        Needed to circumvent compat issues with AdventureWorks2014 db which doesn't
+        support SPLIT_STRING. """
+
+        split_string_func = """CREATE FUNCTION dbo.splitstring ( @stringToSplit VARCHAR(MAX) )\n
```

Comment:
Can't we just run against a 130 compat level or higher? I prefer not workaround this by creating a function, especially it it's a shared/read only database like AdventureWorks.

Reply:
My bad, I completely forgot to make this change. On it now.